samples/ListAzureSubscriptions/index.js

```diff
@@ -0,0 +1,13 @@
+const df = require("durable-functions");
+
+module.exports = df.orchestrator(function*(context) {
+    // More information on the use of managed identities in the callHttp API:
+    // https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-http-features#managed-identities
+    var res = yield context.df.callHttp(
+        "GET",
+        "https://management.azure.com/subscriptions?api-version=2019-06-01",
+        undefined,
+        undefined,
+        new df.ManagedIdentityTokenSource("https://management.core.windows.net"));
+    return res;
```

Comment:
Do you actually want to return the entire response here, or just the body?

Reply:
Ah, yeah, it's probably cleaner to just return the response content. But for now I think I'll leave it as-is. I guess if nothing else it allows people to see the full ARM response, which might be interesting.